base_controller_lib/build.gradle

```diff
@@ -14,17 +14,26 @@
  * the License.
  */
 
+// UsbSerialLibrary is not yet available on Maven, so the path to the artifact
+// has to be sourced to work properly with tools like AndroidStudio.
 dependencies {
-    compile 'org.ros.android_core:android_10:[0.2, 0.3)'
-    compile 'com.hoho.android.usbserial:UsbSerialLibrary:[0.1.1, 0.2)'
+    compile 'org.ros.rosjava_core:rosjava:[0.3.0]'
```

Comment:
I'd say yes, let's do this right. Let's keep this open format and if the change in NativeNodeMain breaks things, we'll release it as a new major number.